### ISSUE TITLE
add rotations

### ISSUE THREAD

**Issue by matthewleon** [haskell/vector#189]
Would PRs to add rotation functions be accepted?

**Comment by cartazio** [comment]
You mean like mod the size of the vector style coordinate shift?

**Comment by matthewleon** [comment]
Yes. Of course it's not hard to do by composing functions that the library already has, but I also find that it's a fairly common thing to want to do with. a vector.
